src/components/transition/test/Slider-test.js

```diff
@@ -24,6 +24,8 @@ describe('Slider', () => {
     container.remove();
   });
 
+  testTransitionComponent(Slider);
+
```

Comment:
Regarding the description of the next test: Does it render _collapsed_ or is it actually just plain old _hidden_?

Reply:
Since this is a Slider-specific test, I thought it made sense to keep the original test descriptions which use the "collapse"/"expand" concepts.

For other TransitionComponents it might indeed not apply.